mmeval/metrics/voc_map.py

```diff
@@ -466,7 +472,7 @@ def compute_metric(self, results: list) -> dict:
             dict: The computed metric, with the following keys:
 
             - mAP, the averaged across all IoU thresholds and all class.
-            - mAP@{IoU}, the mAP of the specified IoU threshold.
+            - AP{IoU}, the mAP of the specified IoU threshold.
```

Comment:
Not familiar with the convention, just a question here.
We denote it as `AP{IoU}`, but display it as `AP50`. Could this be confusing to users, since they passed in `iou_thrs=0.5`?